src/app/routes/keycloakSignInRoutes.js

```diff
@@ -0,0 +1,127 @@
+const bodyParser = require('body-parser');
+const envHelper = require('../helpers/environmentVariablesHelper.js');
+const request = require('request-promise');
+const qs = require('querystring');
+const DOMParser = require('xmldom').DOMParser;
+const parser = new DOMParser();
+const { logger } = require('@project-sunbird/logger');
+const _ = require('lodash');
+
+let formData = {
+  username: "",
+  password: ""
+}
+let redirect_uri = "";
+let client_id = "";
+
+module.exports = (app) => {
+  app.post('/keycloak/login', bodyParser.json(), async (req, res) => {
+    if (req && req.body && req.body.loginConfig) {
```

Comment:
`if (req && req.body && req.body.loginConfig)`
can be written as
`if(_.get(req, 'body.loginConfig')`

Reply:
okay